specification/cognitiveservices/data-plane/Language/stable/2022-05-01/customtextauthoring.json

```diff
@@ -0,0 +1,3000 @@
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The details of the long running operation.",
+            "schema": {
+              "$ref": "#/definitions/TextAnalysisAuthoringImportProjectJobState"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Get Import Status": {
+            "$ref": "./examples/textanalysisauthoring/SuccessfulGetImportStatus.json"
+          }
+        }
+      }
+    },
+    "/authoring/analyze-text/projects/{projectName}/models": {
+      "get": {
+        "description": "Lists the trained models belonging to a project.",
+        "operationId": "TextAnalysisAuthoring_ListTrainedModels",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ProjectNamePathParameter"
+          },
+          {
+            "$ref": "common.json#/parameters/TopParameter"
+          },
+          {
+            "$ref": "common.json#/parameters/SkipParameter"
+          },
+          {
+            "$ref": "common.json#/parameters/MaxPageSizeParameter"
+          },
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "List of all trained models.",
+            "schema": {
+              "$ref": "#/definitions/TextAnalysisAuthoringProjectTrainedModels"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful List Models": {
+            "$ref": "./examples/textanalysisauthoring/SuccessfulListModels.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "value"
+        }
+      }
+    },
+    "/authoring/analyze-text/projects/{projectName}/models/{trainedModelLabel}": {
+      "get": {
+        "description": "Gets the details of a trained model.",
+        "operationId": "TextAnalysisAuthoring_GetTrainedModel",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ProjectNamePathParameter"
+          },
+          {
+            "in": "path",
+            "name": "trainedModelLabel",
+            "description": "The trained model name.",
+            "required": true,
+            "type": "string"
+          },
```

Comment:
Like the other swagger, I imagine this gets reused a lot. At least a local parameter might be nice to make reviews easier.

Reply:
Should we move all the parameters to the parameters section? Or only the re-used ones?

Reply:
I recommend - though not critical - moving commonly used ones. Whether that's 2 or 3+ uses is up to you (or none).